roofit/roofitcore/src/RooNLLVar.cxx

```diff
@@ -459,44 +459,52 @@ Double_t RooNLLVar::evaluatePartition(std::size_t firstEvent, std::size_t lastEv
 /// \param[in] lastEvent   First event not to be processed.
 /// \return Tuple with (Kahan sum of probabilities, carry of kahan sum, sum of weights)
 RooNLLVar::ComputeResult RooNLLVar::computeBatched(std::size_t stepSize, std::size_t firstEvent, std::size_t lastEvent) const
+{
+  auto pdfClone = static_cast<const RooAbsPdf*>(_funcClone);
+  return computeBatchedFunc(pdfClone, _dataClone, _evalData, _normSet, _weightSq, stepSize, firstEvent, lastEvent);
+}
+
+// static function, also used from TestStatistics::RooUnbinnedL
+RooNLLVar::ComputeResult RooNLLVar::computeBatchedFunc(const RooAbsPdf *pdfClone, RooAbsData *dataClone,
```

Comment:
```suggestion
RooNLLVar::ComputeResult RooNLLVar::computeBatchedFunc(const RooAbsPdf & pdf, RooAbsData & data,
```
It's not relevant that these are `*Clone` at this point, and it's better to use references and not pointers if they can't be null.